speech/getting-started/speech_recognition.ipynb

```diff
@@ -0,0 +1,915 @@
+{
```

Comment:
<p>Please add link to documentation. I would also suggest removing the research paper as it's not necessary and is at best distracting to the developer experience in this notebook.</p><p> </p><p>I would also add a little bit more about what's possible with Chirp and why developers should consider using Chirp. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done</p>